images/sqlpad/config/main.tf

```
@@ -0,0 +1,25 @@
+terraform {
+  required_providers {
+    apko = { source = "chainguard-dev/apko" }
+  }
+}
+
+variable "extra_packages" {
+  description = "The additional packages to install"
+  // TODO: Add any other packages here you want to conditionally include,
+  // or update this default to [] if this isn't a version stream image.
+  default = [
+    "sqlpad",
+    "sqlpad-compat"
+    // Other packages your image needs
```

I don't think these comments are still needed.

```suggestion
  default = [
    "sqlpad",
    "sqlpad-compat"
```

---

Sure, will remove them, thanks